seumich/models.py

```diff
@@ -127,7 +127,7 @@ def status_calculated_value(self):
         :return: Calculation of values
         :rtype: float
         """
-        mean_values = [status.code_value for status in self.statuses.all()]
+        mean_values = [status.status.code_value for status in self.studentclasssitestatus_set.all()]
```

Comment:
should it be status.code_value, instead of status.status.code_value?

Reply:
It looks like it might be wrong but "status" is the named variable in the for loop. The object not a Status but StudentClassSiteStatus that is prefetched. I can rename this return variable to make this more clear.

Reply:
I think either rename the variable or add a comment could be helpful.

Reply:
Yeah I pushed a commit that renames it